templates/cli/lib/commands/push.js.twig

```diff
@@ -90,11 +91,13 @@ const awaitPools = {
             return true;
         }
 
-        let steps = Math.max(1, Math.ceil(total / STEP_SIZE));
-        if (steps > 1 && iteration === 1) {
-            pollMaxDebounces *= steps;
+        if (pollMaxDebounces === POLL_DEFAULT_VALUE) {
+            let steps = Math.max(1, Math.ceil(total / STEP_SIZE));
+            if (steps > 1 && iteration === 1) {
+                pollMaxDebounces *= steps;
 
-            log('Found a large number of attributes, increasing timeout to ' + (pollMaxDebounces * POLL_DEBOUNCE / 1000 / 60) + ' minutes')
+                log('Found a large number of attributes, increasing timeout to ' + (pollMaxDebounces * POLL_DEBOUNCE / 1000 / 60) + ' minutes')
+            }
```

Comment:
So we're only going to log if the developer doesn't specify a custom debounce? Why is that?

Reply:
If the developer specified the debounce limit, it will be used, and then it doesn't make sense to notify about it as it will be used anyway.

Reply:
If the developer specified the debounces limit, then there's no need to check if we're increasing the timeout as we won't increase it but set it to the manually one